app/scripts/react-directives/components/modals/modals.jsx

```
@@ -31,20 +31,22 @@ export const Modal = ({ id, active, onCancel, children }) => {
   const ref = useRef();
   useEffect(() => {
     if (active) {
-      const handleClick = event => {
+      const handleClick = (event) => {
         if (ref && ref.current && !ref.current.contains(event.target)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)